stable/siren/values.yaml

```diff
@@ -1,99 +1,122 @@
-replicaCount: 1
-podAnnotations: {}
-labels: {}
-## String to fully override siren.fullname template
-##
-# fullnameOverride:
-## String to fully override siren.name template
-##
-# nameOverride:
+siren-app:
```

Comment:
Need to add an entry for this in the `Chart.yaml -> dependencies`.

Reply:
Oh right, I forgot about this, thanks @rohilsurana